packages/slider/src/mwc-slider-base.ts

```diff
@@ -153,6 +157,27 @@ export class SliderBase extends FormElement {
     }
   }
 
+  updated(changed: PropertyValues) {
+    const minChanged = changed.has('min');
+    const maxChanged = changed.has('max');
+
+    if (minChanged && maxChanged) {
+      if (this.max < this.mdcFoundation.getMin()) {
```

Comment:
I am getting some weird behavior when my min is greater than 100 when both min/max are set. The max gets overwritten.